### PR TITLE
WOC: Adds tooltip hover on top of user input field.

### PR DESCRIPTION
**Pull request by usmanmuhd**
Adds tooltip hover on top of user input field in Advaced payout and basic payout.
Fixes https://github.com/gitcoinco/web/issues/2215
screenshot from 2018-10-05 06-18-37

screenshot from 2018-10-05 06-18-42

whatsapp image 2018-10-04 at 22 01 09

whatsapp image 2018-10-04 at 22 01 09 1

### CODE REVIEW

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 1]
```diff
@@ -120,13 +121,52 @@ <h5>{% trans 'Payout Preview' %}</h5>
                     </a>
                   </div>
                 </div>
+                
+              </div>
+              <div id="tooltiphov">
+                    <span id="heading">Where is my Eth Going?</span><br>
```
Comment (mbeacom) [inline app/dashboard/templates/bulk_payout_bounty.html]: Can you wrap all of the copy throughout this file in `{% trans "copy" %}`

So here:  `{% trans "Where is my Eth Going?" %}`

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 2]
```diff
@@ -120,13 +121,52 @@ <h5>{% trans 'Payout Preview' %}</h5>
...
+                    <strong>Verified Users</strong><br>
+                    We send the funds directly to the address associated with the user.</span>
               </div>
             </div>
           </div>
         </div>
       </div>
     </div>
     <style>
+      #tooltip {
```
Comment (mbeacom) [inline app/dashboard/templates/bulk_payout_bounty.html]: Can we move this CSS to either the top of the template or possibly a separate .css file for bounties specifically?

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 3]
```diff
@@ -201,5 +241,14 @@ <h5>{% trans 'Payout Preview' %}</h5>
   <script src="{% static "v2/js/secrets.min.js" %}"></script>
   <script src="{% static "v2/js/ethereumjs-accounts.js" %}"></script>
   <script src="{% static "onepager/js/send.js" %}"></script>
-
+  <script type="text/javascript">
+    $("#tooltip").hover(
```
Comment (mbeacom) [inline app/dashboard/templates/bulk_payout_bounty.html]: Can this move be to the `bulk_payout.js` ?

File: app/dashboard/templates/process_bounty.html [hunk 4]
```diff
@@ -57,6 +57,7 @@ <h3>{% trans "Basic Payout" %}</h3>
                 </div>
                 <div class="w-100 my-2">
                   <label for=bountyFulfillment>{% trans "Bounty Submission:" %}</label>
+                  <label id="tooltip">Where is my Eth going? <i class='fa fa-info-circle'></i></label>
```
Comment (mbeacom) [inline app/dashboard/templates/process_bounty.html]: Can you wrap all of the copy throughout this file in `{% trans "copy here" %}`

File: app/dashboard/templates/process_bounty.html [hunk 5]
```diff
@@ -103,6 +114,36 @@ <h3>{% trans "Basic Payout" %}</h3>
     document.token_address = '{{bounty.token_address}}';
   </script>
 
+  <style>
+      #tooltip {
```
Comment (mbeacom) [inline app/dashboard/templates/process_bounty.html]: Can we move this to the top or into another css file? cc @thelostone-mc @SaptakS

File: app/dashboard/templates/process_bounty.html [hunk 6]
```diff
@@ -115,4 +156,14 @@ <h3>{% trans "Basic Payout" %}</h3>
   <script src="{% static "v2/js/tokens.js" %}"></script>
   <script src="{% static "v2/js/pages/shared_bounty_mutation_estimate_gas.js" %}"></script>
   <script src="{% static "v2/js/pages/process_bounty.js" %}"></script>
+  <script type="text/javascript">
```
Comment (mbeacom) [inline app/dashboard/templates/process_bounty.html]: Can we move this to the `bulk_payout.js` ?

Comment (usmanmuhd) [reply app/dashboard/templates/process_bounty.html]: `bulk_payout.js` is not included in process_bounty.html. Maybe move it to a completely separate file?

File: app/dashboard/templates/process_bounty.html (hunk 4 above)
Comment (usmanmuhd) [reply app/dashboard/templates/process_bounty.html]: Yeah okay.

File: app/dashboard/templates/bulk_payout_bounty.html (hunk 2 above)
Comment (usmanmuhd) [reply app/dashboard/templates/bulk_payout_bounty.html]: `bounty.css` exists but I can't find that it is included anywhere.

Comment (mbeacom) [reply app/dashboard/templates/bulk_payout_bounty.html]: It's not included in this template, but the majority of our common CSS styles are included via:  `{% include 'shared/head.html' %}` cc @thelostone-mc @SaptakS What path would you like taken here?  Do you want the styles left inline or moved to another asset?

File: app/dashboard/templates/bulk_payout_bounty.html (hunk 1 above)
Comment (thelostone-mc) [reply app/dashboard/templates/bulk_payout_bounty.html]: @usmanmuhd Just the text

File: app/dashboard/templates/bulk_payout_bounty.html (hunk 2 above)
Comment (thelostone-mc) [reply app/dashboard/templates/bulk_payout_bounty.html]: Hmm right now `gitcoin.css` should be alright. that's where I've thrown in the autocomplete css code too

File: app/dashboard/templates/process_bounty.html (hunk 5 above)
Comment (thelostone-mc) [reply app/dashboard/templates/process_bounty.html]: I'd be okay with us moving it to a new file ! let's do that

**Comment by owocki** [comment]
1. the modal is showing on the basic payout, but its not actually true for basic payout.  for basic payout… the funds are sent dirctly to the fulfiller
2. for tips/advanced payout/kudos , that modal should show up.. but the copy is not correct; we never have access to the funds… can we change the copy to say “this secure proxy address will hold the funds until the recipient claims them.  this address is generated by gitcoin’s website, but we dont have your private keys, and the only person who can access the funds is the recipient”  ?

**Comment by thelostone-mc** [comment]
@usmanmuhd My man -> could have the changes in by tom ? would like to ship this asap ^_^

**Comment by usmanmuhd** [comment]
> 1. the modal is showing on the basic payout, but its not actually true for basic payout.  for basic payout… the funds are sent dirctly to the fulfiller
> 2. for tips/advanced payout/kudos , that modal should show up.. but the copy is not correct; we never have access to the funds… can we change the copy to say “this secure proxy address will hold the funds until the recipient claims them.  this address is generated by gitcoin’s website, but we dont have your private keys, and the only person who can access the funds is the recipient”  ?

@owocki I will change it to just verified users on the basic payout page. 
I will also add the tooltip for the tips ?

**Comment by owocki** [comment]
> @owocki I will change it to just verified users on the basic payout page.

'verified users' is not how Basic Payout works.  Basic payout works with [Standard Bounties](https://github.com/Bounties-Network/StandardBounties)

here is some copy you can use:"*Standard Bounties* When you pay via Standard Bounties, your tokens will be securely taken from the escrow [StandardBounties.sol contract on the Ethereum mainnet](https://github.com/Bounties-Network/StandardBounties), and transferred to the payee's wallet"

> I will also add the tooltip for the tips ?

yes pls!

**Comment by usmanmuhd** [comment]
@mbeacom I have made the changes as requested.
@owocki I do not have a design in order to add the tooltip in the tips page. Could you please help with that?

File: app/dashboard/templates/process_bounty.html [hunk 7]
```diff
@@ -85,6 +87,12 @@ <h3>{% trans "Basic Payout" %}</h3>
                   </a>
                 </div>
               </div>
+              <div id="tooltiphov">
+                    <span id="heading">{% trans "Where is my Eth Going?" %}</span><br>
+                    <span id="subheading">{% trans "Your funds are safe!" %}</span><br><br>
+
```
Comment (thelostone-mc) [inline app/dashboard/templates/process_bounty.html]: can we remove the extra line?

File: app/dashboard/templates/process_bounty.html [hunk 8]
```diff
@@ -85,6 +87,12 @@ <h3>{% trans "Basic Payout" %}</h3>
                   </a>
                 </div>
               </div>
+              <div id="tooltiphov">
+                    <span id="heading">{% trans "Where is my Eth Going?" %}</span><br>
+                    <span id="subheading">{% trans "Your funds are safe!" %}</span><br><br>
```
Comment (thelostone-mc) [inline app/dashboard/templates/process_bounty.html]: could we indent by 2 spaces  ?

File: app/assets/v2/css/tooltiphov.css [hunk 9]
```diff
@@ -0,0 +1,28 @@
+#tooltip {
+  color: #4A90E2;
+  float: right;
+}
+#tooltiphov {
```
Comment (thelostone-mc) [inline app/assets/v2/css/tooltiphov.css]: new empty line btwn style declarations

File: app/assets/v2/css/tooltiphov.css [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+  float: right;
+  padding-top: 35px;
+  padding-left: 20px;
+  position: absolute;
+  top: 23rem;
+  left: 535px;
+  z-index: 10;
+  display: none;
+}
+#heading {
```
Comment (thelostone-mc) [inline app/assets/v2/css/tooltiphov.css]: new empty line btwn style declarations

File: app/assets/v2/css/tooltiphov.css [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+  font-size: 20px;
+}
+#subheading {
```
Comment (thelostone-mc) [inline app/assets/v2/css/tooltiphov.css]: new empty line btwn style declarations

File: app/assets/v2/css/tooltiphov.css [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+  font-size: 18px;
+  color: #00A55E;
+}
+#hovertext{
```
Comment (thelostone-mc) [inline app/assets/v2/css/tooltiphov.css]: new empty line btwn style declarations  & add space before bracket. `#hovertext {`

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 13]
```diff
@@ -120,6 +129,17 @@ <h5>{% trans 'Payout Preview' %}</h5>
...
+                
+              </div>
+              <div id="tooltiphov">
+                    <span id="heading">{% trans "Where is my Eth Going?" %}</span><br>
+                    <span id="subheading">{% trans "Your funds are safe!" %}</span><br><br>
+
+                    <span id="hovertext"><strong>{% trans "Unverified Users" %}</strong> <em>{% trans "(no eth address associated)" %}</em><br>
+                    {% trans "A secure proxy address will hold the funds until the recipient claims them. This address is generated by gitcoin’s website, but we don't have its private keys, and the only person who can access the funds is the recipient." %}<br><br>
+
+                    <strong>{% trans "Verified Users" %}</strong><br>
```
Comment (thelostone-mc) [inline app/dashboard/templates/bulk_payout_bounty.html]: Stick to two space indentation

File: app/dashboard/templates/process_bounty.html [hunk 14]
```diff
@@ -115,4 +122,5 @@ <h3>{% trans "Basic Payout" %}</h3>
   <script src="{% static "v2/js/tokens.js" %}"></script>
   <script src="{% static "v2/js/pages/shared_bounty_mutation_estimate_gas.js" %}"></script>
   <script src="{% static "v2/js/pages/process_bounty.js" %}"></script>
+  <script src="{% static "v2/js/tooltiphov.js" %}"></script>
```
Comment (thelostone-mc) [inline app/dashboard/templates/process_bounty.html]: could you rename it to `bounty/tooltip_hover.js` and update the file name ?

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 15]
```diff
@@ -201,5 +221,5 @@ <h5>{% trans 'Payout Preview' %}</h5>
   <script src="{% static "v2/js/secrets.min.js" %}"></script>
   <script src="{% static "v2/js/ethereumjs-accounts.js" %}"></script>
   <script src="{% static "onepager/js/send.js" %}"></script>
-
+  <script src="{% static "v2/js/tooltiphov.js" %}"></script>
```
Comment (thelostone-mc) [inline app/dashboard/templates/bulk_payout_bounty.html]: Same as [here](https://github.com/gitcoinco/web/pull/2334/files#r222685955)

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 16]
```diff
@@ -20,6 +20,14 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
+    <link rel="stylesheet" type="text/css" href="{% static "v2/css/tooltiphov.css" %}">
```
Comment (thelostone-mc) [inline app/dashboard/templates/bulk_payout_bounty.html]: could you rename it to bounty/tooltip_hover.css and update the file name ?

File: app/assets/v2/css/tooltiphov.css (hunk 9 above)
Comment (thelostone-mc) [reply app/assets/v2/css/tooltiphov.css]: rename this to `tooltip--hover` and update references ?

**Comment by thelostone-mc** [comment]
@usmanmuhd Left a few comments! Once that's done -> could you post a screenshot of it looks on mobile too ?

**Comment by usmanmuhd** [comment]
@thelostone-mc I have made the necessary changes. As for mobile, I am afraid `hover` does not work.

**Comment by usmanmuhd** [comment]
@thelostone-mc Added the screenshots for mobile as well.

**Comment by thelostone-mc** [comment]
@usmanmuhd updated CSS a tad bit 👍 Thanks for turning this around quick :) 

@owocki / @PixelantDesign we are good for payout. 

x

**Comment by usmanmuhd** [comment]
Shall I submit this on gitcoin?

**Comment by owocki** [comment]
this is great!  can we get it added on the tips page too?  https://github.com/gitcoinco/web/issues/2366

File: app/dashboard/templates/bulk_payout_bounty.html [hunk 17]
```diff
@@ -20,6 +20,33 @@
   <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
+    <link rel="stylesheet" type="text/css" href="{% static "v2/css/tooltip_hover.css" %}">
+    <style type="text/css">
```
Comment (owocki) [inline app/dashboard/templates/bulk_payout_bounty.html]: is there a reason this css is inline instead of in `tooltip_hover.css`?

Comment (usmanmuhd) [reply app/dashboard/templates/bulk_payout_bounty.html]: It is basically reusing the `#tooltip--hover` design primarily designed keeping `process_bounty.html` in mind, and making a few changes to it.

**Pull request merged**